metrics/web/test/common/presentation/navigation/state/navigation_notifier_test.dart

```diff
@@ -167,13 +168,91 @@ void main() {
       },
     );
 
+    test(
+      ".handleLoggedOut() clears pages when a user logs out",
+      () {
+        notifier.handleLoggedIn();
+        notifier.push(DefaultRoutes.dashboard);
+
+        notifier.handleLoggedOut();
+
+        expect(notifier.pages, hasLength(1));
+      },
+    );
+
+    test(
+      ".handleLoggedOut() pushes to the login page when a user logs out",
+      () {
+        notifier.handleLoggedIn();
+        notifier.push(DefaultRoutes.dashboard);
+
+        notifier.handleLoggedOut();
+
+        final currentPage = notifier.pages.last;
```

Comment:
🔨 
```suggestion
        final currentPage = notifier.currentConfiguration;
```
Does it make sense?